packages/frontend/hooks/src/use-block-suite-page-preview.ts

```diff
@@ -60,7 +60,7 @@ export function useBlockSuitePagePreview(page: Page | null): Atom<string> {
         page.slots.ready.on(() => {
           set(getPagePreviewText(page));
         }),
-        page.slots.yUpdated.on(() => {
```

Comment:
Will block content changes trigger the blockUpdated event?

Reply:
I will try to use page indexer to get all references.